e2e-tests/js_scripts/0008-update-runtime-multisig-scheduler.js

```diff
@@ -0,0 +1,153 @@
+     *****************************************************************************************************/
+    const threshold = 2;
+    const multisigAddress = util.createKeyMulti([alice.address, bob.address, charlie.address], threshold);
+    const SS58Prefix = 42;
+    const Ss58MultiAddress = util.encodeAddress(multisigAddress, SS58Prefix);
+    console.log(`multisigAddress ${Ss58MultiAddress}`);
+
+
+    /*****************************************************************************************************
+     *************************************** SET MULTISIG AS SUDO ACCOUNT ********************************
+     *****************************************************************************************************/
+
+    const newSudoKey = api.tx.sudo.setKey(Ss58MultiAddress);
+    await submitExtrinsic(newSudoKey, alice, BlockUntil.Finalized, undefined);
+
+    const newSudoKeyOption = await api.query.sudo.key();
+    if (newSudoKeyOption.isSome && newSudoKeyOption.unwrap().toString() !== Ss58MultiAddress) {
+        return ReturnCode.ErrWrongSudoAccount;
+    }
+
+    /*****************************************************************************************************
+     *************************************** GET CURRENT RUNTIME VERSION******* **************************
+     *****************************************************************************************************/
+
+    const currentRuntimeVersion = (await api.rpc.state.getRuntimeVersion()).specVersion.toNumber();
+
+    /*****************************************************************************************************
+     *************************************** SCHEDULE RUNTIME UPGRADE*************************************
+     *****************************************************************************************************/
+
+    // Retrieve the runtime to upgrade
+    const sudoAccount = await api.query.sudo.key()
+    const code = fs.readFileSync('./new-runtime/nh_runtime.wasm').toString('hex');
+    const updateRuntimeCall = api.tx.system && api.tx.system.setCode
+        ? api.tx.system.setCode(`0x${code}`) // For newer versions of Substrate
+        : api.tx.consensus.setCode(`0x${code}`); // For previous versions
+    const sudoUpdateRuntimeCall = api.tx.sudo.sudo(updateRuntimeCall)
+
+    console.log(`Upgrading from ${sudoAccount}, ${code.length / 2} bytes`);
+
+    const { block } = await api.rpc.chain.getBlock();
+    const blockNumber = block.header.number.toNumber();
+
+    const when = blockNumber + 10;
+    const priority = 100;
+
+    const scheduleTx = api.tx.scheduler.schedule(when, null, priority, sudoUpdateRuntimeCall);
+    const paymentInfo = await scheduleTx.paymentInfo(alice);
+
+    const proposal = api.tx.multisig.asMulti(
+        threshold,
+        [BOB, CHARLIE].sort(),
+        null,
+        scheduleTx.method.toHex(),
+        0
+    );
+
+    await submitExtrinsic(proposal, alice, BlockUntil.Finalized, undefined);
+
```

Comment:
```suggestion
    await submitExtrinsic(proposal, alice, BlockUntil.InBlock, undefined);
```
We don't need to wait the finalized one in test